features/configuration_files/masking_smells.feature

```diff
@@ -22,28 +22,34 @@ Feature: Masking smells using config files
     Then it succeeds
     And it reports nothing
 
-  Scenario: allow masking some calls for duplication smell
+  Scenario: provide extra masking inline in comments
```

Comment:
Can you put these scenarios in their original order? I'm having a hard time seeing what changed here.